dev_guide/builds.adoc

```diff
@@ -2028,16 +2028,13 @@ manually)
 [[build-resources]]
 == Build Resources
 
-A build is completed by a pod that consumes resources (memory and CPU) on a
-node. By default, pods consume unbounded node resources. However, if a project
-specifies default container limits, then pods consume resources up to those
-limits.
+When a pod completes a build, unbound resources, such as memory and CPU, are
```

Comment:
I think the original phrasing of the first sentence might be better here.  not all pods execute builds, so saying "when a pod completes a build" implies that one of the steps of running a pod is for it to complete a build, which is not true.